packages/astro/test/fixtures/astro-doctype/src/pages/capital.astro

```diff
@@ -0,0 +1,9 @@
+---
+let title = 'My Site';
+---
+
+<!DOCTYPE html>
```

Comment:
With Astro 0.17.3, when using only a `PageLayout.astro ` to define the basic HTML document structure for every pages, the `doctype ` is not injected when missing, and is removed when manually added.
In other words, the /dist/ index.html does not contain the doctype in any of these situations.

[Demo project with live preview link:](https://github.com/alex-drocks/portfolio-alex-astro)

in /layouts/PageLayout.astro I tried:

```html
<html lang="en">
<head>
...
```

and tried manually adding doctype:
```html
<!DOCTYPE html> // or <!doctype html>
<html lang="en">
<head>
...
```